main.cpp

```diff
@@ -73,12 +23,26 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void ToString() {
+    std::cout<<"REFERENCE MANUAL OF NUMBER TO COLOR MAPPING"<<std::endl;
+    std::cout<<" "<<"Number"<<" "<<"Major Minor"<<std::endl;
+    
+    int Number = 1;
+    while(Number<=25){
```

Comment:
Do not hardcode array limits like 25. Almost surely someone will forget to update this while adding a new color.